src/Alpacka.Lib/Instances/MultiMC/MultiMCHandler.cs

```diff
@@ -48,11 +53,13 @@ public void Install(string instancePath, ModpackBuild pack)
         public void Update(string instancePath, ModpackBuild oldPack, ModpackBuild newPack)
         {
             var multiMCInstancePath = Path.GetDirectoryName(instancePath);
+            var multiMCInstanceCfg = Path.Combine(multiMCInstancePath, CONFIG_FILE);
             
             // update minecraft and forge version
-            if (newPack.MinecraftVersion != oldPack?.MinecraftVersion)
-                MultiMCInstance.UpdateVersion(multiMCInstancePath, newPack.MinecraftVersion, newPack.ForgeVersion);
-            
+            var cfg = new MultiMCConfig(multiMCInstanceCfg);
```

Comment:
or would oyu want ot check multiple times and read, parse, write multiple times ?